lib/Controllers/TeamController.php

```diff
@@ -35,6 +37,8 @@ public function contributor(string $github) : Response
     {
         $contributor = $this->contributorRepository->findOneByGithub($github);
 
+        assert($contributor instanceof Contributor);
```

Comment:
Move the assertion into the repository method, and then you can make the return value there non-nullable

Reply:
Done.